### PR TITLE
Suggest page5

### PR DESCRIPTION
**Pull request by julianecasier**
New release of SuggestPage, using remarks of old PR and working!

### CODE REVIEW

**Review by Elfaram — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/assets/datasuggest.js [hunk 1]
```diff
@@ -14,53 +14,62 @@ export default [
...
     picture: imgYoung,
     alt: "young",
+    query: "release_date.lte=2010-01-01&release_date.gte=1990-01-01",
   },
   {
     key: "baby",
     type: "dateRelease",
     picture: imgBaby,
     alt: "baby",
+    query: "release_date.gte=2010-01-01&release_date=2010-01-01",
```
Comment (LoicBrassart) [inline frontend/src/assets/datasuggest.js]: Nope, won't give the right results

File: frontend/src/components/QuestionSug/index.jsx [hunk 2]
```diff
@@ -1,66 +1,113 @@
...
+  });
+
+  const handleChoiceLinks = (evt) => {
+    evt.target.classList.toggle("selected");
+    const key = evt.target.name;
+    setFormData({ ...formData, [key]: !formData[key] });
+  };
+
+  const makeQuestSubmit = () => {
+    const allTags = Object.keys(formData); // object.keys gets all keys of an object and puts it into an array
```
Comment (LoicBrassart) [inline frontend/src/components/QuestionSug/index.jsx]: It's `Object.keys()` (the capital O is **important**)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fixes to do !

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by Elfaram — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
NmE!

**Review by Elfaram — DISMISSED** [review]

**Comment by julianecasier** [comment]
It was validated but merging had to be solved... I did it on my computer and it has to be checked again...

**Review by Elfaram — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**